### PR TITLE
fix: mobile ui sidebar and sentence review

### PR DESCRIPTION
**Pull request by alejandroclose**
Fixed changes requested and changed the size of buttons and text box to adapt it better to small screens.

### CODE REVIEW

**Comment by MichaelKohler** [comment]
Thanks for the PR, I will review this beginning of next week.

File: web/css/review-form.css [hunk 1]
```diff
@@ -8,13 +8,14 @@
 }
 
 .sentence-box {
-  height: 6rem;
+  height: 9.25rem;
```
Comment (MichaelKohler) [inline web/css/review-form.css]: this and the change below in the CSS seem to have negative effects, there is quite some overlap now:

Screenshot 2019-08-14 at 18 38 32

Comment (alejandroclose) [reply web/css/review-form.css]: @MichaelKohler I believe its fixed now! I only updated the `validator-height` class that is a css variable. 
I'm not able to test it with more than one sentence as I can't review the sentences I add. Could you guide me to how I could add more mock sentences? Thank you!! 

Let me know what you think! :)

File: web/css/review-form.css [hunk 2]
```diff
@@ -1,5 +1,36 @@
+/* media queries */
+
+@media screen and (min-width: 1025px) {
+  .validator{
```
Comment (MichaelKohler) [inline web/css/review-form.css]: I haven't tried, but I think you could leave the variables in the normal definitions, and only overwrite it in the `max-width: 1024px` media query. I think that would be cleaner. The same would go for the other definitions.

File: web/css/review-form.css [hunk 3]
```diff
@@ -1,5 +1,36 @@
...
+   --sentence-box-font-size: var(--base-font-size);
+ }
+
+ .validator .button-group button {
+  --button-padding: .7rem 1.3rem;
+}
+}
+
+@media screen and (max-width: 1024px) {
+  .validator{
```
Comment (MichaelKohler) [inline web/css/review-form.css]: minor: could you add a space before the `{`?

File: web/css/review-form.css [hunk 4]
```diff
@@ -1,5 +1,36 @@
...
+    --validator-height: 10.5rem;
+   }
+
+  .sentence-box{
```
Comment (MichaelKohler) [inline web/css/review-form.css]: minor: could you add a space before the `{`?

**Comment by alejandroclose** [comment]
@MichaelKohler Amazing!! Thank you!!! Ready for the next task :) 
What do you think I should do? Continue iterating the mobile UI/UX or it would add more value if I look for another issue and try to fix it?

**Comment by MichaelKohler** [comment]
> @MichaelKohler Amazing!! Thank you!!! Ready for the next task :)
> What do you think I should do? Continue iterating the mobile UI/UX or it would add more value if I look for another issue and try to fix it?

Would https://github.com/Common-Voice/sentence-collector/issues/258 be of interest to you?

**Comment by alejandroclose** [comment]
> > @MichaelKohler Amazing!! Thank you!!! Ready for the next task :)
> > What do you think I should do? Continue iterating the mobile UI/UX or it would add more value if I look for another issue and try to fix it?
> 
> Would #258 be of interest to you?

Yes!!! I'll start working on it! :)

**Pull request merged**